stl/inc/system_error

```diff
@@ -121,7 +121,7 @@ _NODISCARD inline bool _System_error_equal(const error_code&, const error_condit
 // CLASS error_code
 class error_code { // store an implementation-specific error code and category
 public:
-    error_code() noexcept : _Myval(0), _Mycat(&system_category()) { // construct non-error
+    error_code() noexcept : _Myval(0), _Mycat(&_STD system_category()) { // construct non-error
     }

```

Comment:
Since we're here:
```suggestion
    error_code() noexcept : _Myval(0), _Mycat(&_STD system_category()) {} // construct non-error
```

Reply:
I went ahead and cleaned up almost all of the braces. You asked for it! 😹